src/saml20_clj/crypto.clj

```diff
@@ -27,6 +27,13 @@
    :excl-omit-comments SignatureConstants/ALGO_ID_C14N_EXCL_OMIT_COMMENTS
    :excl-with-comments SignatureConstants/ALGO_ID_C14N_EXCL_WITH_COMMENTS})
 
+(defn has-private-key? [credential]
```

Comment:
This should have a docstring since it's a public function (I guess this convention wasn't being used in this repo, but might as well start)